constructor/nsis/main.nsi.tmpl

```diff
@@ -137,6 +137,8 @@ BrandingText /TRIMLEFT "${COMPANY}"
 #!define MUI_PAGE_CUSTOMFUNCTION_SHOW OnStartup
 !define MUI_PAGE_CUSTOMFUNCTION_PRE SkipPageIfUACInnerInstance
 !insertmacro MUI_PAGE_WELCOME
+# Custom welcome file(s)
+@CUSTOM_WELCOME_FILE@
```

Comment:
My proposal was that `custom_welcome` would _replace_ `MUI_PAGE_WELCOME`. The replacement code can include the macro again if you want to keep it, but this gives users a chance to actually remove the default Welcome page.

(See related comment in `winexe.py`)

```suggestion
#!define MUI_PAGE_CUSTOMFUNCTION_SHOW OnStartup
#if custom_welcome
# Custom welcome file(s)
@CUSTOM_WELCOME_FILE@
#else
!define MUI_PAGE_CUSTOMFUNCTION_PRE SkipPageIfUACInnerInstance
!insertmacro MUI_PAGE_WELCOME
#endif
```